omod/src/main/java/org/openmrs/module/fhir/providers/RestfulEncounterResourceProvider.java

```diff
@@ -68,6 +71,18 @@ public Encounter getResourceById(@IdParam IdDt theId) {
 		return encounterResource.searchEncountersById(id);
 	}
 
+	/**
+	 * Search encounters by patient identifier
+	 *
+	 * @param identifier object containing the patient identifier
+	 */
+	@Search()
+	public List<Encounter> searchEncountersByPatientIdentifier(
+		@RequiredParam(name=Encounter.SP_PATIENT, chainWhitelist= {Patient.SP_IDENTIFIER}) ReferenceParam identifier
```

Comment:
In that case dynamic chain usage is fine. Looks good.

Can you update the documentation as well?